README.md

```diff
@@ -1,2 +1,74 @@
-# Capstone1
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
```

Comment:
- [ ] Kindly add a descriptive description of your project to your README so that other people know more about your project.